### DISCUSSION TITLE
brew install gitlab on macos failed

### DISCUSSION THREAD

**Discussion by liyuerich** [Homebrew/discussions#4510] [Getting started] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.18
ORIGIN: https://github.com/Homebrew/brew
HEAD: b0dc84b117f54f6b18e0528b95477c54e409ebdd
Last commit: 4 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: d2b110ec2585c7d2c81f1db9def3f87bfceed552
Core tap last commit: 7 days ago
Core tap branch: master
Core tap JSON: 17 May 01:13 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: octa-core 64-bit icelake
Clang: 13.0.0 build 1300
Git: 2.30.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.77.0 => /usr/bin/curl
macOS: 12.0.1-x86_64
CLT: 13.0.0.0.1.1627064638
Xcode: N/A
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences.

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 14.2.


Warning: You have an unnecessary local Core tap!
This can cause problems installing up-to-date formulae.
Please remove it by running:
 brew untap homebrew/core
```

### Description of issue

brew install gitlab

Warning: No available formula with the name "gitlab". Did you mean glab?
==> Searching for similarly named formulae and casks...
==> Formulae
gitlab-ci-local                   gitlab-gem                        gitlab-runner                     glab ✔

To install gitlab-ci-local, run:
  brew install gitlab-ci-local

==> Casks
github

To install github, run:
  brew install --cask github

**Comment by liyuerich** [comment]
try to install gitlab at macos, get the message:

Warning: No available formula with the name "gitlab". Did you mean glab?

**Reply by SMillerDev** [reply]
Yes, no such package exists. What are you trying to do?

**Reply by liyuerich** [reply]
do you know how to get this package?

**Reply by SMillerDev** [reply]
There is no such package. There is nothing to get.

**Reply by carlocab** [reply]
Maybe you're looking for the `glab` package? Try `brew install glab`.

**Reply by liyuerich** [reply]
the background is I want to run a shell at my Mac, however an error message said "gitlab: command not found"

then I try to install gitlab at my Mac by using  brew install gitlab

**Reply by ZhongRuoyu** [reply, marked as answer]
Were you looking for `gitlab-gem`? It provides a `gitlab` executable. You may install it with

```
brew install gitlab-gem
```

**Reply by liyuerich** [reply]
thanks your suggestion, it does works.
